apps/filecoin-site/src/app/_components/SimpleCard.tsx

```diff
@@ -0,0 +1,66 @@
+import clsx from 'clsx'
+
+import { Badge, type BadgeProps } from '@/components/Badge'
+import { CTALink, type CTALinkProps } from '@/components/CTALink'
+import { Heading } from '@/components/Heading'
+
+export type SimpleCardProps = {
+  title: string
+  description: string
+  cta: {
+    href: CTALinkProps['href']
+    text: CTALinkProps['children']
+  }
+  gradient?: {
+    from: string
+    to: string
+  }
+  badge?: {
+    text: BadgeProps['children']
+  }
+  hasTopBorder?: boolean
+}
+
+export function SimpleCard({
+  title,
+  description,
+  cta,
+  gradient,
+  badge,
+  hasTopBorder = false,
+}: SimpleCardProps) {
+  const isGradientCard = gradient && badge
+  const isBorderedCard = hasTopBorder
+
+  return (
+    <li
+      className={clsx(
+        'border border-black/15',
```

Comment:
I know Figma uses a color + transparency value, but could we use the zinc scale here instead? (for consistency)

If Filipa agrees of course 😄


```suggestion
        'border border-zinc 200',
```